core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.html

```diff
@@ -683,6 +676,27 @@
     padding: 32px;
     width: 388px;
   }
+  .learn-dropdown-menu .nav-item-left:hover,
+  .learn-dropdown-menu .nav-item-right:hover,
+  .learn-dropdown-menu .nav-item-left:focus,
+  .learn-dropdown-menu .nav-item-right:focus {
+    background-color: #21212107;
```

Comment:
Can we use a hex color code here? ditto else where

Reply:
PTAL!